main.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-use-before-define */
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const btn = document.getElementById('btn-add');
+
+function BookConstructor(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function storeBooks(books) {
+  return localStorage.setItem('books', books);
+}
+
+function getBooks() {
+  let books = JSON.parse(localStorage.getItem('books'));
+  if (!books) {
+    books = [];
+    return books;
+  }
+  return books;
+}
+
+let arrBooks = getBooks();
+
+function addBooks() {
+  const title = bookTitle.value.toString();
+  const author = bookAuthor.value.toString();
+  const book = new BookConstructor(title, author);
+  const inputs = document.querySelectorAll('input');
+  inputs.forEach((inputs) => {
+    inputs.value = '';
+  });
+  arrBooks.push(book);
+  storeBooks(JSON.stringify(arrBooks));
+  displayBooks();
+}
+
+function createDisplay(bookContent) {
+  arrBooks.map((e, i) => {
+    bookContent += `<p class="title">${arrBooks[i].title}</p>
+  <p class="author">${arrBooks[i].author}</p>
+  <button class="remove">Remove</button>
+  <div class="line"></div>`;
+    document.querySelector('.books').innerHTML = bookContent;
+    const buttons = document.querySelectorAll('button.remove');
+    const removeBook = (index) => {
+      arrBooks = arrBooks.filter((e, i) => i !== index);
+      storeBooks(JSON.stringify(arrBooks));
+      // eslint-disable-next-line no-use-before-define
+      return displayBooks();
+    };
+    buttons.forEach((e, i) => {
+      buttons[i].addEventListener('click', (event) => {
+        // eslint-disable-next-line no-undef
```

Comment:
- [ ] Please avoid disabling ESlint to get around linter warnings problems. These ESlint warnings are there to ensure you adhere to correct coding standards and when you have warnings they usually signal an underlying issue with how you've written your code.